src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -235,7 +235,7 @@ export default class StyleConfigurator {
     style[mxConstants.STYLE_STROKEWIDTH] = 1.5;
     style[mxConstants.STYLE_ROUNDED] = 1;
     style[mxConstants.STYLE_ARCSIZE] = 5;
-    style[mxConstants.STYLE_VERTICAL_ALIGN] = mxConstants.ALIGN_BOTTOM;
+    style[mxConstants.STYLE_VERTICAL_ALIGN] = mxConstants.ALIGN_TOP;
```

Comment:
:question: why changing this? If the visualization tests are not impacted, this means this options is useless or we need more visualization tests.
Restoring the initial value breaks the refactored model tests

Reply:
I am currently checking our tests but it seems that we don't have visualization tests for default label positions.

Reply:
Confirmed, this breaks the rendering (original on top of this diff image, rendering with the current change on the bottom)

![bpmn-rendering-test-ts-no-visual-regression-labels-03-default-position-1-diff](https://user-images.githubusercontent.com/27200110/95847441-00ae5500-0d4d-11eb-84a9-18bb4b839ca8.png)

Reply:
I am currently adding the tests in #749

Reply:
I change the value in order to harmonize the value and simplify the test configuration. sometimes it's 'top' (when there is label bound. i'm not sure this value is use in this case), sometimes it's 'bottom' (no label bounds).

Reply:
After testing, if I change the STYLE_VERTICAL_ALIGN value when there is label bounds, the label position changes 👎 
So I'll revert my modification, and I update the configuration of the tests.